.github/workflows/test-hawk.yaml

```diff
@@ -0,0 +1,51 @@
+name: e2e_test for hawk
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+env:
+  IP_NODE1: 172.17.0.2
+  IP_NODE2: 172.17.0.3
+
+jobs:
+  functional_test_crm_report_bugs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Start cluster nodes in containers
+        run: |
+          docker run -d --privileged -h node1 --name node1 --add-host node1:${IP_NODE1} --add-host node2:${IP_NODE2} ghcr.io/aleksei-burlakov/hawk-node
```

Comment:
I would try to upload this image somewhere else: `ghcr.io/aleksei-burlakov`
Otherwise, if somebody wants/needs to make a change, it is locked to your ghcr registry

Reply:
> I don't really understand why do you use specific containers, and not openSUSE leap for instance

It's because the hawk is compiled inside the container and this requires different gems. We actually practice that the private images are used to test, e.g. https://github.com/ClusterLabs/hawk/blob/master/test/Dockerfile#L1 or in the crmsh https://github.com/ClusterLabs/crmsh/blob/master/test/run-functional-tests#L3 .
I will add the Dockefile in the next PRs. In this PR I want to add at least a simple test.